task/01-strings-tasks.js

```diff
@@ -201,7 +201,10 @@ function extractEmails(str) {
  *
  */
 function getRectangleString(width, height) {
-    throw new Error('Not implemented');
+    const topLine = '┌' + '─'.repeat(width - 2) + '┐\n';
+    const middleLine = '│' + ' '.repeat(width - 2) + '│\n';
+    const bottomLine = '└' + '─'.repeat(width - 2) + '┘\n';
+    return topLine + middleLine.repeat(height - 2) + bottomLine;
```

Comment:
There is a [good practice](https://eslint.org/docs/latest/rules/padding-line-between-statements) to add a new line before the `return` line